### PR TITLE
Add Nodle network, types file to dev

### PR DESCRIPTION
**Pull request by leohar**

### CODE REVIEW

File: chains/types/nodle.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"runtime_id": 1, "types": {"Balance": "u128", "Index": "u32", "Phase": "frame_system.Phase", "Address": "sp_runtime.multiaddress.MultiAddress", "ExtrinsicSignature": "sp_runtime.MultiSignature", "ParaId": "polkadot_parachain.primitives.Id", "runtime_eden.Event": "GenericEvent", "runtime_eden.Call": "GenericCall", "sp_core.crypto.AccountId32": "GenericAccountId", "pallet_identity.types.Data": "Data"}, "versioning": []}
```
Comment (antonkhvorov) [inline chains/types/nodle.json:1]: Move to types/v2

Comment (stepanLav) [inline chains/types/nodle.json:1]: Let's make the json prettier by formatting.

File: chains/v2/chains_dev.json [hunk 2]
```diff
@@ -2510,5 +2510,41 @@
         "color": "linear-gradient(315deg, #007568 31.86%, #91E3DA 100%)",
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Integritee.svg",
         "addressPrefix": 13
+    },
+    {
+        "chainId": "97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21",
+        "name": "Nodle Parachain",
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:2516]: Just "Nodle"

File: chains/v2/chains_dev.json [hunk 3]
```diff
@@ -2510,5 +2510,41 @@
...
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "NODL",
+                "precision": 11,
+                "priceId": "nodle",
```
Comment (antonkhvorov) [inline chains/v2/chains_dev.json:2522]: It's seems it's not listed yet, so I would remove that one as well

File: chains/v2/chains_dev.json [hunk 4]
```diff
@@ -2510,5 +2510,41 @@
         "color": "linear-gradient(315deg, #007568 31.86%, #91E3DA 100%)",
         "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Integritee.svg",
         "addressPrefix": 13
+    },
+    {
+        "chainId": "97da7ede98d7bad4e36b4d734b6055425a3be036da2a332ea5a7037656427a21",
```
Comment (stepanLav) [inline chains/v2/chains_dev.json:2515]: Need to add "parentId" for polkadot parachain it is:
"parentId": "91b171bb158e2d3848fa23a9f1c25182fb8e20313b2c1eb49219da7a70ce90c3",

**Review by antonkhvorov — CHANGES_REQUESTED** [review]

File: chains/types/nodle.json (hunk 1 above)
Comment (leohar) [reply chains/types/nodle.json]: done

Comment (leohar) [reply chains/types/nodle.json]: fixed

File: chains/v2/chains_dev.json (hunk 2 above)
Comment (leohar) [reply chains/v2/chains_dev.json]: done

File: chains/v2/chains_dev.json (hunk 3 above)
Comment (leohar) [reply chains/v2/chains_dev.json]: done

File: chains/v2/chains_dev.json (hunk 4 above)
Comment (leohar) [reply chains/v2/chains_dev.json:2515]: fixed

**Review by stepanLav — APPROVED** [review]

**Review by antonkhvorov — APPROVED** [review]

**Comment by valentunn** [comment]
Please attach screenshot of Polkadot.js information page next time so each reviewer wont need to go check it

**Pull request merged**